includes/events/event-builder.php

```diff
@@ -987,8 +987,18 @@ private function get_attachments($attachments)
 		foreach ($attachments as $attachment) {
 			$html .= '<li class="simcal-attachment">';
 			$html .= '<a href="' . $attachment['url'] . '" target="_blank">';
-			$html .= !empty($attachment['icon']) ? '<img src="' . $attachment['icon'] . '" />' : '';
-			$html .= '<span>' . $attachment['name'] . '</span>';
+			if (isset($attachment['mime']) && strpos($attachment['mime'], 'image') !== false) {
```

Comment:
Do test if it does not run on core plugin since we can't have drive access there